lca8s8-d1e737/cod-dlc9qq_lca8s8-d1e737.xml

```diff
@@ -150,16 +150,16 @@
           christi <ref xml:id="lca8s8-d1e737-Rd1e311" target="http://scta.info/resource/Icor9_12">1a Corinth. 9</ref> Numquid 
           <lb ed="#P" n="6"/>pro se solo quasi solus teneretur euange<lb ed="#P" break="no" n="7"/>lium 
           gratis dare: et non potius pro omni<lb ed="#P" break="no" n="8"/>bus 
-          illud dixit quod in <ref xml:id="lca8s8-d1e737-Rd1e320">actibus aposto<lb ed="#P" break="no" n="9"/>lorum 
+          illud dixit quod in <ref xml:id="lca8s8-d1e737-Rd1e320" target="http://scta.info/resource/act20_33-35">actibus aposto<lb ed="#P" break="no" n="9"/>lorum 
```

Comment:
If this ref is for the with the source, then all we need is to link the ref to the quote. I use the corresp attribute for that like so:
```suggestion
          illud dixit quod in <ref xml:id="lca8s8-d1e737-Rd1e320" corresp="#lca8s8-d1e737-Qd1e344">actibus aposto<lb ed="#P" break="no" n="9"/>lorum 
```